aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -214,7 +104,7 @@ T quantize_val(double scale, int64_t zero_point, float value) {
 
 template <typename T, int precision>
 void quantize_vec(double scale, int64_t zero_point, const float *src, T *dst, size_t count) {
```

Comment:
why are these not moved to `quant_affine`?
Also please use a better name for the file, e.g. affine_quantizer

Reply:
Good point, I think you are right. Your naming provides a better picture of the dependencies. Quantization functions share some checks, so leave them in one file to keep things simple for now.
About quantize_val and other things - I guess it is a good point. Didn't move before to keep PR simple - it is hard to track diff between files.